pkg/apis/kf/v1alpha1/space_defaults.go

```diff
@@ -45,10 +45,16 @@ func (k *SpaceSpec) SetDefaults(ctx context.Context, name string) {
 	k.ResourceLimits.SetDefaults(ctx)
 }
 
+const DefaultBuildServiceAccountName = "kf-build-creds"
```

Comment:
I think it's weird to have the service account called "creds", maybe just "kf-build"?

Reply:
Sounds good. I'll change it to "kf-builder"